New impure (#2778)
modelica/ModelicaSpecification: chapters/functions.tex

allowed.
\end{nonnormative}

> Only if the function formal parameter is \lstinline!impure!, it is allowed to pass an \lstinline!impure! function to it, due to \cref{function-compatibility-or-function-subtyping-for-functions}.

Reviewer: Maybe this is one of those places where the plain text formatted _impure_ would work even better than `impure`?  There's a small chance that it could reduce the risk of being interpreted as the `impure` being part of the formal parameter's component declaration…

Reviewer: With some rephrasing to be more direct, it could look like this:
```suggestion
By \cref{function-compatibility-or-function-subtyping-for-functions}, it follows that an impure function can only be passed as argument to a function formal parameter of impure type.
```
